src/handbook/product/plan.md

```diff
@@ -36,14 +36,14 @@ The progress bars presented adjacent to each category are rough estimates, offer
 
 
 ### Node-RED Deployments at the Edge (Device Management)
-![](https://geps.dev/progress/40)
+![](https://geps.dev/progress/80)
 
 - [Device can be managed independently of Instances on Application level](https://github.com/FlowFuse/flowfuse/issues/2334)
 
 - [Grouped Device Deployments](https://github.com/FlowFuse/flowfuse/issues/1509)
 
 ### Enhanced Integration Capabilities
-![](https://geps.dev/progress/60)
+![](https://geps.dev/progress/95)
```

Comment:
Can we update the link on line 48 to go to `/certified-nodes`? And I believe that custom NPM Modules are more work than 5% remaining in this category